### ISSUE TITLE
[ApolloCodegen] Support for multiple headers

### ISSUE THREAD

**Issue by LachlanMcCulloch** [apollographql/apollo-ios#1131]
Currently, in `ApolloCodegen` you can only initialize `ApolloSchemaOptions` with a single `header` property, which is translated to `--header="<HEADER>"`. Since the Apollo CLI allows for multiple headers arguments (`... --header=<HEADER1> --header=<HEADER2>`), I was wondering if it would be possible to implement this in this tool?

One potential way, albeit a breaking change, is to do the following:
```swift
let headers: [String]
...
  public init(schemaFileName: String = "schema",
              schemaFileType: SchemaFileType = .json,
              apiKey: String? = nil,
              endpointURL: URL,
              headers: [String] = [],
              outputFolderURL: URL,
              downloadTimeout: Double = 30.0) {
    self.apiKey = apiKey
    self.headers = headers
    self.endpointURL = endpointURL
    self.outputURL = outputFolderURL.appendingPathComponent("\(schemaFileName).\(schemaFileType.rawValue)")
    	
    self.downloadTimeout = downloadTimeout
  }

...
for header in headers {
    arguments.append("--header='\(header)'")
}
```

This changes header to an array of headers instead and provides the same logic to append extra headers to the end of the CLI call.

I've tested this with a forked repo and it appears to still work, of course after updating any implementations to use `headers`, instead of `header`:
https://github.com/LachlanMcCulloch/apollo-ios/commit/fe5d417c332327d51170ea7bf2a327870c136b4f

Let me know if you need any more supporting information or clarification.

Cheers! :)

**Comment by designatednerd** [comment]
Yep this is exactly why this is still in Beta. 😃

Wanna open a PR?

**Comment by LachlanMcCulloch** [comment]
@designatednerd - sure can!
https://github.com/apollographql/apollo-ios/pull/1153

**Comment by designatednerd** [comment]
Awesome, thank you!

**Comment by LachlanMcCulloch** [comment]
Thanks @designatednerd 😄

**Comment by designatednerd** [comment]
This has shipped with `0.26.0`!
